.github/workflows/backend_master.yml

```diff
@@ -93,8 +93,8 @@ jobs:
         CONFIG_PATH=examples/k8s/backend/config.yaml
         echo "$BE_CONFIG_DATA" | base64 --decode > $CONFIG_PATH
 
-        sed -i 's|#{be.imageTag}#|'"$IMAGE_TAG"'|' examples/k8s/backend/kustomization.yaml
-        sed -i 's|#{be.imageTag}#|'"$IMAGE_TAG"'|' examples/k8s/backend/be.yaml
+        sed -i '' 's|#{be.imageTag}#|'"$IMAGE_TAG"'|' examples/k8s/backend/kustomization.yaml
```

Comment:
Why do you need additional whitespace in all the `sed` invocations?

Reply:
it's a small fix for being able to run deployment scripts on macos.
https://stackoverflow.com/questions/7573368/in-place-edits-with-sed-on-os-x